test/tests/unit/override/adopted-probes.test.ts

```diff
@@ -412,16 +416,14 @@ describe('AdoptedProbes', () => {
 
 		expect(sql.raw.callCount).to.equal(2);
 
-		expect(sql.raw.args[0]![1]).to.deep.equal({
+		expect(sql.raw.args[0]![1]).to.deep.include({
 			recipient: '3cff97ae-4a0a-4f34-9f1a-155e6def0a45',
-			subject: `Your probe's location has changed`,
-			message: 'Globalping detected that your [probe with IP address **1.1.1.1**](/probes/p-1) has changed its location from Ireland to United Kingdom. The custom city value "Dublin" is not applied anymore.\n\nIf this change is not right, please report it in [this issue](https://github.com/jsdelivr/globalping/issues/268).',
+			subject: 'Your probe\'s location has changed',
 		});
 
-		expect(sql.raw.args[1]![1]).to.deep.equal({
+		expect(sql.raw.args[1]![1]).to.deep.include({
 			recipient: '3cff97ae-4a0a-4f34-9f1a-155e6def0a45',
 			subject: `Your probe's location has changed`,
-			message: 'Globalping detected that your probe [**probe-gb-london-01**](/probes/p-9) with IP address **9.9.9.9** has changed its location from Ireland to United Kingdom. The custom city value "Dublin" is not applied anymore.\n\nIf this change is not right, please report it in [this issue](https://github.com/jsdelivr/globalping/issues/268).',
```

Comment:
The formatting here is rather tricky so having a test for it doesn't seem like a bad idea, why remove it?

Reply:
I though it is excess work to fix tests on every message update. And since it is .md format, update will still require manual check.

Reply:
It's some extra work, but it's very easy to break the message, especially since there are multiple variants of it. That's why each variant got included in the tests as well.

Reply:
Ok, I'll get it back.